app/dummy_feature/routes.py

```diff
@@ -0,0 +1,46 @@
+from flask import flash, redirect, render_template, url_for
+from flask_login import current_user, login_required
+
+from app import db
+from app.dummy_feature import bp
+from app.dummy_feature.forms import RoomForm
+from app.dummy_feature.models import Room
+
+# Main page.
+@bp.route('/dummy_page')
+@login_required
+def dummy_page():
+    # Retrieve rooms.
+    rooms = Room.query.all()
```

Comment:
Variable "rooms" is rendered to the template, but it's not checked, if rooms is null, HTML will be crash.